dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/util/RequestUtils.java

```diff
@@ -636,7 +680,12 @@ static public Hashtable parseQueryString(char[][] cha, String encoding) {
         }                                                                                            //PK75617
 
         if (cha == null || cha.length == 0) {
-            throw new IllegalArgumentException("query string or post data is null");
+            if (SERVLET61_OR_ABOVE) {
+                throw new IllegalStateException("query string or post data is null");
```

Comment:
Should this be translated?

Reply:
it is currently not translated in the service.

2 Options here:
(1). Translate with 6.1+ and keep older version  as-is.
(2). Translate regardless of servlet version.  If any customer is relying on the current message, it may break them.  

NOTE that the ISE is thrown in 6.1+ VS IAE in <6.1. If anyone is relying solely on just the message, they may not see the difference.

Reply:
Thanks for adding the translation!